Common/UnitDefinitions/CoefficientOfThermalExpansion.json

```diff
@@ -32,13 +34,59 @@
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "°C⁻¹", "1/°C" ]
+          "Abbreviations": []
```

Comment:
We can't remove the existing abbreviations. 

Did you run into trouble with tests failing on ambiguous units when parsing these abbreviations?

I guess an improvement to avoid ambiguity when renaming units would be to add support for specifying in JSON that this unit cannot be parsed, so the abbreviations are only used for ToString().

Maybe something like `"SkipParse": true` or declaring a separate `"ParseAbbreviations": []` to override abbreviations for parse. The latter one is more flexible.

What do you think?

Reply:
I believe what we have done in the past is to keep one of the least used abbreviations in the old unit, and all other abbreviations in the new unit. In this case we could keep `1/°C` here, and move  `°C⁻¹` to the new unit, but it's not a very good solution.

Reply:
In all honesty I haven't really paid much attention to the abbreviation functionality in what I have been using the library for. Your workaround seems reasonable to me for now, but I guess your proposed solution seems preferable.

The reason you would want to skip it during parsing is because the unit is obsolete, so perhaps an `"IsObsolete" : true`?

For now shall I implement the workaround pending a better solution via a different issue?

Reply:
I'm fine with the workaround